benchmark/Cpp/Savina/src/concurrency/BoundedBuffer.lf

```diff
@@ -0,0 +1,227 @@
+/**
+ * Copyright (C) 2020 TU Dresden
+ * 
+ * This benchmark models a bounded buffer that is used to synchronize between a
+ * group of consumers and a group of producers. In the Akka implementation of
+ * this benchmark, all producers, consumers and the manager operate fully
+ * asynchronously. However, in the LF implementation everything is synchronous.
+ * Thus, strictly speaking, the buffer isn't actually needed as we can design
+ * the program so that every tick a producer produces data and a consumer
+ * consumes it. But, the number of consumers and producers can be seleced
+ * arbitrarily. Therefore, we still need some logic that takes care of or
+ * orchestrating the producers according to the rate at which the consumers
+ * consume. Also, we could implement the consumer such that they don't become
```

Comment:
I don't understand the "could" in this sentence.  Should it be "must"?

Reply:
No, I wanted to say that the benchmark could also be designed differently. I rephrased the sentence to make this clearer (I hope).